src/main/java/com/pp/demo/UserRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.pp.demo;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class UserRepository {
+
+    private List<User> userList = new ArrayList<>();
```

Comment:
`final` dorzuć :)

Reply:
Tu nadal finala brakuje :P

Reply:
Jak robie finala to dostaje:
Cannot assign a value to final variable 'userList' w konstruktorze ....